README.md

```diff
@@ -367,7 +367,8 @@ values (EG: "default_server": True)
                                 "share_adjusters": [
                                     {
                                         "share_adjuster_class": "aurproxy.share.adjusters.RampingShareAdjuster",
```

Comment:
I don't feel strongly about it either way, and am open to changing it. I think that it ended up nested in order to make it easy for the unit test module and application module to share a common python module (which in turn made nose configuration easier).